pages/common/vhs.md

```diff
@@ -0,0 +1,24 @@
+# vhs
+
+> Generate Terminal Gifs from Code.
+> More information: <https://github.com/charmbracelet/vhs>.
+
+- Record inputs to a tape file. Once done, exit the shell to create the tape:
+
+`vhs record > {{tape_name}}`
+
+- Record inputs to a tape file using a specific shell:
+
+`vhs record --shell {{shell}} > cassette.tape`
+
+- Validate a type file's syntax:
+
+`vhs validate {{tape_name_or_path}}`
+
+- Create gif from tape File:
+
+`vhs < {{tape_name_or_path}}.tape`
```

Comment:
```suggestion
`vhs < {{path/to/file.tape}}`
```
I think we can have the extension inside the placeholder, to be consistent throughout the page even though it is the only working one.